xwiki-platform-core/pom.xml

```diff
@@ -398,6 +398,19 @@
                     "old": "parameter void org.xwiki.notifications.filters.watch.WatchedUserReference::<init>(java.lang.String, org.xwiki.notifications.filters.internal.user.EventUserFilterPreferencesGetter, ===org.xwiki.notifications.filters.NotificationFilterManager===)",
                     "new": "parameter void org.xwiki.notifications.filters.watch.WatchedUserReference::<init>(java.lang.String, org.xwiki.notifications.filters.internal.user.EventUserFilterPreferencesGetter, ===org.xwiki.notifications.filters.NotificationFilterPreferenceManager===)",
                     "justification": "Young API"
+                  },
+                  {
```

Comment:
You might want to remove that and build to get a more lightweight version from the new revapi version error. In any case you should definitely remove newArchive which is going to fail the release.

Reply:
Ok I'll fix that, thanks!